usaspending_api/etl/transaction_loaders/field_mappings_fpds.py

```diff
@@ -33,7 +39,6 @@
     "ultimate_parent_unique_ide": "ultimate_parent_unique_ide",
     "award_description": "award_description",
     "place_of_performance_zip4a": "place_of_performance_zip4a",
-    "place_of_performance_zip5": "place_of_performance_zip5",
```

Comment:
I don't think we want to change these "raw" fields in `transaction_fpds`, it's similar to the whitespace discussion.

Reply:
Is that generally the case for all the derived fields in `transaction_fpds_functions`? What about the other tables that use these values? Are we alright with them being different?

Reply:
> Is that generally the case for all the derived fields in `transaction_fpds_functions`?

These are different than the existing fields being altered.

> What about the other tables that use these values? Are we alright with them being different?

It would require research on where these columns are used and how they are used.  It might be necessary to maintain the current loader behavior but it should be investigated first and  determined to be necessary.